scripts/prepare-machine.ps1

```diff
@@ -109,17 +109,6 @@ if ($PSVersionTable.PSVersion.Major -lt 7) {
                  "Please visit https://github.com/microsoft/msquic/blob/main/docs/BUILD.md#powershell-usage")
 }
 
-if ($UseXdp) {
-    # Helper for XDP usage
-    if ($ForBuild) {
-        $InstallXdpSdk = $true;
```

Comment:
Why skipping installing XdpSdk for build?

Reply:
`if ($ForBuild) {` below already has `$InstallXdpSdk = $true`.